gsmgogo-api/src/main/java/team/gsmgogo/domain/match/service/impl/MatchServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+        List<MatchEntity> matches = matchQueryDslRepository.findByMonthAndDay(month, day);
+        List<MatchResultEntity> matchResults = matchResultQueryDslRepository.findByMonthAndDay(month, day);
+        List<BetEntity> bettings = betJpaRepository.findByUser(currentUser);
+
+        List<MatchInfoDto> matchList = matches.stream()
+            .filter(match -> match.getEndAt().isBefore(LocalDateTime.now()))
+            .map(match -> {
+                return new MatchInfoDto(
+                    match.getMatchId(),
+                    match.getMatchType(),
+                    match.getMatchLevel(),
+                    match.getTeamA().getTeamId(),
+                    match.getTeamA().getTeamName(),
+                    match.getTeamAGrade(),
+                    match.getTeamAClassType(),
+                    match.getTeamB().getTeamId(),
+                    match.getTeamB().getTeamName(),
+                    match.getTeamBGrade(),
+                    match.getTeamBClassType(),
+                    match.getStartAt(),
+                    match.getEndAt(),
+                    bettings.stream().anyMatch(bet -> bet.getMatch() == match),
+                    match.getTeamABet(),
+                    match.getTeamBBet()
+                );
+            }).toList();
+
+        List<MatchResultDto> endedMatches = matchResults.stream()
+            .map(matchResult -> {
+                MatchEntity match = matchResult.getMatch();
+                Optional<BetEntity> betting = bettings.stream()
+                    .filter(bet -> bet.getMatch() == match)
+                    .findFirst();
+
+                CalculatePointRequest request = new CalculatePointRequest(
+                    betting.get().getBetPoint(), 
+                    matchResult.getTeamAScore(), 
+                    matchResult.getTeamBScore(), 
+                    betting.get().getBetScoreA(), 
+                    betting.get().getBetScoreB(), 
+                    match.getTeamABet(), 
+                    match.getTeamBBet()
+                );
+                CalculatePointResponse calculatePoint = new CalculatePoint().execute(request);
+
+                return new MatchResultDto(
+                    match.getMatchId(), 
+                    match.getMatchType(), 
+                    match.getMatchLevel(), 
+                    match.getTeamA().getTeamId(), 
+                    match.getTeamA().getTeamName(), 
+                    match.getTeamAGrade(), 
+                    match.getTeamAClassType(), 
+                    match.getTeamB().getTeamId(), 
+                    match.getTeamB().getTeamName(), 
+                    match.getTeamBGrade(), 
+                    match.getTeamBClassType(), 
+                    betting.isPresent(), 
+                    match.getTeamABet(), 
+                    match.getTeamBBet(), 
+                    matchResult.getTeamAScore(), 
+                    matchResult.getTeamAScore(), 
```

Comment:
```suggestion
                    matchResult.getTeamBScore(), 
```

B팀 점수인데 A팀 점수가 들어가네요!